src/help/components/landing/styles/help-quick-search.module.scss

```diff
@@ -0,0 +1,42 @@
+@import 'franklin-sites/src/styles/colours';
+
+.help-autocomplete {
+  &__result-list {
+    max-height: 30rem;
+    overflow-y: auto;
+  }
+
+  &__all-link {
+    text-align: right;
+  }
+
+  :global(.search-input__suffix) {
+    color: $colour-help-green;
+  }
+
+  :global(input[type='text']) {
+    margin-bottom: 0.5rem;
+  }
+
+  :global(.card) {
+    width: initial;
+    margin-top: 0;
+  }
+
+  :global(.info-list) {
+    margin: 0rem;
+  }
+
+  :global(.decorated-list-item) {
+    padding: 0.5rem;
+  }
+
+  :global(.decorated-list-item__title) {
+    flex-basis: 25vw;
```

Comment:
<img width="1015" alt="Screenshot 2021-09-23 at 08 18 10" src="https://user-images.githubusercontent.com/6025561/134462192-fa1b1716-162c-4bab-bef7-7d3f7785da41.png">
This gives strange layout when the screen is wide, the screenshot is for very wide screen.
I'd say use `25%` instead, so that it stays with a basis of 25% of the width of container

Reply:
Yeah, looks nicer. Can you think of a way to get the infolists's separating bar to have the height of `max(height title, height content)`?
<img width="1110" alt="Screenshot 2021-09-23 at 08 37 30" src="https://user-images.githubusercontent.com/4357962/134470357-5306e6ac-2358-4816-a373-38686388c3e9.png">

Reply:
I would do `align-self: stretch` on `:global(.decorated-list-item__content)` a bit lower in the same file

Reply:
Nice! I think this is the intended style so we might as well put this in franklin

Reply:
If you put that in franklin have a look if you can have a more generic rule than this override. using `align-items: stretch` on the flex container instead of the specific child for example. And what out for side effects in different use cases (thinking specifically of the 2 columns approaches)

Reply:
I can't see any unwanted side-effects so pushed this to franklin's main.